blocks/form/constant.js

```diff
@@ -17,7 +17,7 @@ export const defaultErrorMessages = {
 };
 
 // eslint-disable-next-line no-useless-escape
-export const emailPattern = '([A-Za-z0-9][_.\\-]?)+[A-Za-z0-9]@[A-Za-z0-9]+(\.?[A-Za-z0-9]){2}\.([A-Za-z0-9]{2,4})?';
+export const emailPattern = '([A-Za-z0-9][_.\\-]?)+[A-Za-z0-9]@[A-Za-z0-9\.]+(\.?[A-Za-z0-9]){2}\.([A-Za-z0-9]{2,4})?';
```

Comment:
can you use Gen AI for validation as it gave following issues in pattern

Issues in Your Regex:
	1.	Allows Consecutive Periods (..) in the Domain:
	•	Your regex allows multiple dots in the domain (e.g., test@domain..com), which is invalid.
	2.	Incorrect TLD Limitation ({2,4}):
	•	Limits TLDs to 2-4 characters (com, org, net work fine, but .technology, .solutions won’t be valid).
	•	Modern domains like .museum, .aero, and .travel will fail.
	3.	Incorrect Placement of Optional Characters in Local Part ([_.\-]?):
	•	Allows an email to end with _, ., or - (e.g., testuser.@domain.com), which is invalid.
	4.	Does Not Support Newer TLDs & Domain Extensions:
	•	The regex does not correctly enforce modern TLD rules.

Reply:
@jalagari I tried all the negative examples in this list with the current pattern and all were flagged as invalid email.